cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js

```diff
@@ -0,0 +1,75 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Configurations } from '../../../support/fragments/settings/remote-storage';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
+import RemoteStorage from '../../../support/fragments/settings/remote-storage/remoteStorage';
+
+let user;
+const name = `AutotestConfigurationName${getRandomPostfix()}`;
+const caiaSoft = Configurations.configurations.CaiaSoft;
+const dematicStagingDirector = Configurations.configurations.DematicStagingDirector;
+
+describe('remote-storage-configuration', () => {
+  before('create test data', () => {
+    cy.createTempUser([Permissions.remoteStorageCRUD.gui]).then((userProperties) => {
+      user = userProperties;
+      cy.login(user.username, user.password);
+    });
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C367964 Verify text of success toast when creating remote storage configurations (firebird) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+    () => {
+      // #1 Go to the "Settings" app
+      cy.visit(TopMenu.settingsPath);
+      // #2 Select "Remote storage" by clicking on it
+      SettingsMenu.openRemoteStorageSettings();
```

Comment:
combine step 1 and 2 into cy.visit(SettingsMenu.remoteStoragePath)

Reply:
fixed